package.json

```diff
@@ -45,66 +46,68 @@
     }
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
-    "babel-core": "^6.18.2",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.2.7",
-    "babel-plugin-istanbul": "^4.0.0",
-    "babel-plugin-transform-class-properties": "^6.22.0",
-    "babel-plugin-transform-es2015-classes": "^6.24.1",
-    "babel-plugin-transform-es5-property-mutators": "^6.24.1",
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-register": "^6.23.0",
-    "chai": "^3.5.0",
-    "copy-webpack-plugin": "^4.3.0",
-    "cross-env": "^3.1.4",
-    "css-loader": "^0.28.4",
-    "documentation": "^8.1.2",
-    "eslint": "^3.10.0",
-    "eslint-loader": "^1.6.1",
-    "eslint-plugin-flowtype": "^2.30.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-mocha-no-only": "^0.0.5",
-    "eslint-plugin-prettier": "^2.6.2",
-    "flow-bin": "latest",
-    "husky": "^0.14.3",
+    "@babel/cli": "^7.10.1",
+    "@babel/core": "^7.10.2",
+    "@babel/plugin-proposal-class-properties": "^7.10.1",
+    "@babel/plugin-proposal-object-rest-spread": "^7.10.1",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-classes": "^7.10.1",
+    "@babel/plugin-transform-flow-strip-types": "^7.10.1",
+    "@babel/plugin-transform-property-mutators": "^7.10.1",
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-flow": "^7.10.1",
+    "@babel/register": "^7.10.1",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "chai": "^4.2.0",
+    "copy-webpack-plugin": "6.0.3",
+    "cross-env": "^7.0.2",
+    "css-loader": "^3.6.0",
+    "documentation": "^13.0.1",
+    "eslint": "^7.3.1",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-flowtype": "^5.1.4",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-mocha-no-only": "^1.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "flow-bin": "^0.128.0",
+    "husky": "^4.2.5",
     "istanbul": "^0.4.5",
-    "karma": "^1.5.0",
+    "karma": "^5.1.0",
     "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-cli": "^1.0.1",
-    "karma-coverage": "^1.1.1",
-    "karma-firefox-launcher": "^1.0.1",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-cli": "^2.0.0",
+    "karma-coverage": "^2.0.2",
+    "karma-firefox-launcher": "^1.3.0",
     "karma-ie-launcher": "^1.0.0",
-    "karma-mocha": "^1.3.0",
+    "karma-mocha": "^2.0.1",
     "karma-safari-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^2.0.2",
-    "lint-staged": "^7.2.0",
-    "mocha": "^3.2.0",
+    "karma-webpack": "^4.0.2",
+    "lint-staged": "^10.2.11",
+    "mocha": "^8.0.1",
     "mocha-cli": "^1.0.1",
-    "playkit-js-env-manager": "https://github.com/kaltura/playkit-js-env-manager.git#v3.3.2",
```

Comment:
isn't it needed anymore?

Reply:
I don't believe someone using it (unfortunately...) but it was wrongly deleted. thanks